src/components/dashboard/Dashboard.js

```diff
@@ -231,14 +231,15 @@ const Dashboard = props => {
           // a flag used to show feed load animation only at the first app loading
           //subscribeToFeed calls this method on mount effect without dependencies because currently we dont want it re-subscribe
           //so we use a global variable
+
```

Comment:
which is a purpose of this change ?

Reply:
@johnsmith-gooddollar previously store.set('feedLoadAnimShown')(true) was never called